components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/saml/SAML2BearerGrantHandlerTest.java

```diff
@@ -260,6 +265,7 @@ public void testValidateGrantException(Object dateTimeObj, String idpName, boole
         when(userRealm.getUserStoreManager()).thenReturn(userStoreManager);
         when(userStoreManager.isExistingUser(anyString())).thenReturn(isUserExist);
 
+
```

Comment:
Remove redundant line

Reply:
Addressed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1438